API/DTOs/Person/PersonAliasCheckDto.cs

```diff
@@ -0,0 +1,17 @@
+namespace API.DTOs;
+
+public class PersonAliasCheckDto
+{
+    /// <summary>
+    /// The person to check against
+    /// </summary>
+    public int PersonId { get; set; }
```

Comment:
Use [Required] to signal to the api generator that this is required. Users complained.